docs/en/core-services/Transformation.md

```diff
@@ -22,7 +22,7 @@ There are two kinds of supported transformations:
 1. coalesce the files so that each file is at least a certain size and there will be a maximum of certain number of files.
 2. convert CSV files to Parquet files
 
-> In Alluxio version 2.3.0, the transformed data is always written in Parquet format.
+> In Alluxio version 2.5.0, the transformed data is always written in Parquet format.
```

Comment:
I don't know from which version, but can we do something like `2.2 and later`?

Reply:
I don't know from which version so I'll just stick to 2.2.0 and later.